backend/user/src/controllers/questions/index.ts

```diff
@@ -0,0 +1,38 @@
+import { StatusCodes } from 'http-status-codes';
+import type { IRouteHandler } from '@/types';
+import { getAttemptedQuestionsService, addAttemptedQuestionService } from '@/services/questions';
+
+export const addAttemptedQuestion: IRouteHandler = async (req, res) => {
+  const userId = req.params.userId; // Assuming the userId is passed as a route parameter
```

Comment:
can consider having a list of user ids to assist batch requests by interview room on FE